docs/developer-guide/shareable-configs.md

```diff
@@ -4,7 +4,11 @@ The configuration that you have in your `.eslintrc` file is an important part of
 
 ## Creating a Shareable Config
 
-Shareable configs are simply npm packages that export a configuration object. To start, [create a Node.js module](https://docs.npmjs.com/getting-started/creating-node-modules) like you normally would. Make sure the module name begins with `eslint-config-`, such as `eslint-config-myconfig`. Create a new `index.js` file and export an object containing your settings:
+Shareable configs are simply npm packages that export a configuration object. To start, [create a Node.js module](https://docs.npmjs.com/getting-started/creating-node-modules) like you normally would. Make sure the module name begins with `eslint-config-`, such as `eslint-config-myconfig`.
+
+npm [scoped modules](https://docs.npmjs.com/misc/scope) are also supported, by naming or prefixing the module with `@scope/eslint-config`, such as `@scope/eslint-config` or `@scope/eslint-config-myconfig`.
+
+Create a new `index.js` file and export an object containing your settings:
```

Comment:
That's true, however this instruction was already there. I guess it is done that way because of the way the npm creation tutorial works, it guides you through the creation of a module which "main" file is "index.js".  (The tutorial is linked at the beginning of this guide).

Let me know what you think about this.